middlewares/validation.js

```diff
@@ -0,0 +1,28 @@
+const { createError } = require('~/utils/errorsHelper')
+const { BODY_IS_NOT_DEFINED } = require('~/consts/errors')
+const { validateRequired, validateType, validateLength } = require('~/utils/validationHelper')
+
+const validationMiddleware = (schema) => {
+  return function validation(req, _res, next) {
+    try {
```

Comment:
don't use try catch, here we have sync code which can be handled by default express handler